diff --git a/makefile b/makefile
index 6513518..ba14638 100644
--- a/makefile
+++ b/makefile
@@ -85,10 +85,10 @@ $(title).gb: $(objlisto)
 	$(RGBFIX) -jvsc -k "OK" -l 0x33 -m ROM -p 0xFF -t "LIBBET" -v $@
 
 obj/gb/%.o: src/%.z80 src/hardware.inc src/global.inc
-	${RGBASM} -h -o $@ $<
+	${RGBASM} -o $@ $<
 
 obj/gb/%.o: obj/gb/%.z80
-	${RGBASM} -h -o $@ $<
+	${RGBASM} -o $@ $<
 
 # Files that will be included with incbin
 
@@ -108,7 +108,7 @@ obj/gb/sgb.o: \
 
 # Local variable allocation
 
-obj/gb/localvars.z80: tools/savescan.py $(wildcard src/*.z80)
+obj/gb/localvars.z80: tools/savescan.py $(sort $(wildcard src/*.z80))
 	$(PY) $^ -o $@
 
 # Graphics conversion
